crates/iota/src/name_commands.rs

```diff
@@ -201,16 +219,35 @@ impl NameCommand {
         self,
         context: &mut WalletContext,
     ) -> Result<NameCommandResult, anyhow::Error> {
+        let iota_client = context.get_client().await?;
+
         Ok(match self {
             Self::Auction(AuctionCommand::Bid {
                 domain,
                 amount,
                 coin,
                 opts,
             }) => {
-                let auction_package_address = get_auction_package_address(context).await?;
+                let graphql_client = SimpleClient::new(
+                    context
+                        .active_env()?
+                        .graphql()
+                        .as_ref()
+                        .ok_or_else(|| anyhow::anyhow!("missing graphql url in IotaEnv"))?,
+                );
+                let auction_package_address = get_auction_package_address(&iota_client).await?;
                 let auction_house_id =
-                    get_auction_house_id(auction_package_address, context).await?;
+                    get_auction_house_id(auction_package_address, &graphql_client).await?;
+                let auction_house =
+                    get_object_from_bcs::<AuctionHouse>(&iota_client, auction_house_id).await?;
```

Comment:
Use `get_auction_house` instead of the previous lines?

Reply:
I need to auction_house_id :c

Reply:
Oh I see